test/command/command.go

```diff
@@ -20,11 +20,18 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
 )
 
+// RunOptions allows additional options
+type RunOptions struct {
+	IsOutputExpected bool
+	ExecutePod       *corev1.Pod
+}
+
 // RunCommandInCluster runs a command in a pod in the cluster and returns the output
-func RunCommandInCluster(ctx context.Context, c *kubernetes.Clientset, nodeName string, ns string, command string, log logr.Logger) (string, error) {
+func RunCommandInCluster(ctx context.Context, c *kubernetes.Clientset, nodeName string, ns string, command string, log logr.Logger, opts ...*RunOptions) (string, error) {
```

Comment:
Please update the function description, since now the output is not always returned.

Reply:
Good point, however it should be explained in the docstring, together with the fact that the function doesn't actually takes multiple `RunOptions` (IIUC)

Reply:
> together with the fact that the function doesn't actually takes multiple RunOptions

I think this is an excellent point, I've modified the code so it'll take multiple RunOptions, which I think is the standard pattern.

>it should be explained in the docstring

done